config/spinnaker.yml

```diff
@@ -231,8 +231,8 @@ services:
 
   stackdriver:
     enabled: ${SPINNAKER_STACKDRIVER_ENABLED:false}
-    projectName: ${SPINNAKER_STACKDRIVER_PROJECT_NAME:${providers.google.primaryCredentials.project}}
```

Comment:
I *think* you could do this with `${SPINNAKER_STACKDRIVER_PROJECT_NAME:#{providers.google.primaryCredentials.project}}`